monitoring/prometheus.go

```diff
@@ -16,11 +16,8 @@ var (
 	// serverMetrics is a global variable that holds the Prometheus metrics
 	// for the gRPC server.
 	serverMetrics *grpc_prometheus.ServerMetrics
-)
 
-const (
-	// dbTimeout is the default database timeout.
-	dbTimeout = 20 * time.Second
```

Comment:
Let's keep the constant and rename it to `defaultTimeout`? Then we don't have to have the magic number below.

Reply:
Sure, makes sense